src/git-pull-request

```diff
@@ -2,25 +2,15 @@
 source "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/helpers/helpers.sh"
 
 
-# Returns the source for a new Bitbucket pull request
-function bitbucket_source {
-  sha=$(git log --format="%H" -1 | cut -c-12)
-  echo "$repository:$sha:$branch" | sed 's/\//%2F/g' | sed 's/\:/%3A/g'
+function preconditions {
+  activate_driver_family 'code_hosting' \
+                         'Pull requests can only be created on GitHub or Bitbucket.'
```

Comment:
I agree. That wasn't possible before because the error message was command-specific, but I came up with a generic one that is good enough.